W-15468372-HPA-GA-Docs-Fix (#197)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ch2-configure-horizontal-autoscaling.adoc

In Kubernetes, a Horizontal Pod Autoscaler (HPA) automatically updates a workload resource to scale the workload to match demand. Horizontal scaling automatically deploys more pods as a response to an increased load. For more information, visit the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale[Kubernetes documentaton^]. 

> Applications configured with Horizontal Pod Autoscaling are not limited by the xref:general::pricing.adoc#global-resource-pool-limits[Global Resource Pool Limit].

Reviewer: ```suggestion
The xref:general::pricing.adoc#global-resource-pool-limits[Global Resource Pool Limit] does not limit the applications configured with Horizontal Pod Autoscaling.
```
